### ISSUE TITLE
Tutorial 2 raises exception on Mac OS X 10.13.6.

### ISSUE THREAD

**Issue by sighduck1** [beeware/toga#755]
## Context
I followed the tutorial instructions and encountered the following uncaught exception when running tutorial 2 with `python -m app` (`app.py` contained the contents of the tutorial 2 code):

```
  File "/Users/dev/Desktop/.../app.py", line 91, in build
    shortcut=toga.Key.MOD_1 + 'k',
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/enum.py", line 326, in __getattr__
    raise AttributeError(name) from None
AttributeError: MOD_1
```

In tutorial 0, the documentation states to install toga with the following command `pip install --pre toga`. This installs `toga` version `0.3.0.dev14`, which doesn't have  `MOD_1` in [`toga.keys.Key`](https://github.com/beeware/toga/blob/v0.3.0.dev14/src/core/toga/keys.py). In the master branch, [`toga.keys.Key.MOD_1`](https://github.com/beeware/toga/blob/master/src/core/toga/keys.py#L91) exists.

I believe https://github.com/beeware/toga/issues/742 might be referring to the same issue. Sorry if this issue is redundant or if I made a mistake here.

## Your Environment

* Python Version: 3.6.8

* Operating System and Version (select from the following and list the specific version number; if your OS is not listed, list that as well)

  - [x] macOS - version: 10.13.6 (High Sierra)
  - [ ] Linux - distro:   - version:
  - [ ] Windows - version:
  - [ ] Other - name:  - version:

* Toga Version (list the specific version number or git hash)
```
> pip freeze
rubicon-objc==0.3.1
toga==0.3.0.dev14
toga-cocoa==0.3.0.dev14
toga-core==0.3.0.dev14
travertino==0.1.2
```

* Toga Target (the type of app you are trying to generate)

  - [ ] android
  - [x] cocoa
  - [ ] django
  - [ ] gtk
  - [ ] iOS
  - [ ] tvOS
  - [ ] watchOS
  - [ ] winforms
  - [ ] win32
  - [ ] Other (please specify)

**Comment by freakboy3742** [comment]
@sighduck1 Thanks for the report, and no - it's not redundant. I noticed a similar problem myself a day or two ago when testing something, but I got distracted and neglected to log a bug report, and I didn't catch that the problem might have been what was reported by #742 

From some quick testing, I think that the problem has been resolved in master, so cutting a new release will address the issue. I'll do some more comprehensive testing to confirm.
